### DISCUSSION TITLE
How do you pronounce the name?

### DISCUSSION THREAD

**Discussion by HubKing** [ihhub/fheroes2#6951] [Q&A] [answered]
Is it "eff heroes"?

**Comment by LeHerosInconnu** [comment, marked as answer]
Hello @HubKing,

> Is it "eff heroes"?

Enable audio:

https://user-images.githubusercontent.com/43583994/230703566-917604c7-1121-4453-97a5-e3d07e2151cc.mp4

**Reply by HubKing** [reply]
Doesn't it sound like "F*** you, heroes"?

https://www.dictionary.com/browse/eff

**Reply by oleg-derevenetz** [reply]
> Doesn't it sound like "F*** you, heroes"?

You say that like it's something bad.

**Reply by Roxterat** [reply]
More like F heroes 2, I'm making my own engine! *drama music plays*
